Mathlib/AlgebraicGeometry/Morphisms/FinitePresentation.lean

```diff
@@ -71,4 +74,88 @@ instance {X Y Z : Scheme.{u}} (f : X ⟶ Z) (g : Y ⟶ Z) [LocallyOfFinitePresen
     LocallyOfFinitePresentation (Limits.pullback.snd f g) :=
   MorphismProperty.pullback_snd _ _ inferInstance
 
+instance {R : Type*} [CommRing R] : PrespectralSpace (PrimeSpectrum R) :=
+  .of_isTopologicalBasis PrimeSpectrum.isBasis_basic_opens
+    (by simpa using PrimeSpectrum.isCompact_basicOpen)
+
+instance {X : Scheme.{u}} : PrespectralSpace X :=
+  have (Y : Scheme.{u}) (_ : IsAffine Y) : PrespectralSpace Y :=
+    .of_isInducing (Y := PrimeSpectrum _) (Y.isoSpec.hom.base)
+      Y.isoSpec.hom.homeomorph.isInducing ((quasiCompact_iff_spectral _).mp inferInstance)
+  have (i) : PrespectralSpace (X.affineCover.map i).opensRange.1 :=
+    this (X.affineCover.map i).opensRange (isAffineOpen_opensRange (X.affineCover.map i))
+  .of_isOpenCover X.affineCover.isOpenCover_opensRange
+
+def Scheme.Hom.isoOpensRange {X Y : Scheme} (f : X.Hom Y) [IsOpenImmersion f] :
```

Comment:
```suggestion
def Scheme.Hom.isoOpensRange {X Y : Scheme} (f : X ⟶ Y) [IsOpenImmersion f] :
```
no?

Reply:
No. The dot notation wouldn't work.

Reply:
Have you tried? I am pretty sure it works since `X ⟶ Y := X.Hom Y`

Reply:
Dot notation wouldn't know that.

Reply:
Have you tried?

Reply:
Yes.